charts/linkerd2-cni/values.yaml

```diff
@@ -13,7 +13,7 @@ proxyControlPort: 4190
 # -- Docker image for the CNI plugin
 cniPluginImage:   "cr.l5d.io/linkerd/cni-plugin"
 # -- Tag for the CNI container Docker image
-cniPluginVersion: linkerdVersionValue
+cniPluginVersion: "1.0.0"
```

Comment:
I wonder if as part of this change it would be worth following a similar values format as we do for the controller, i.e

```
image:
  name: "cr.l5d.io/linkerd/cni-plugin"
  version: "1.0.0"
  imagePullPolicy: 
```

@alpeb might have some thoughts.

Reply:
Yeah, it'd be nice to have the same struct as elsewhere, with the caveat of the version as described above.